### PR TITLE
Varios Updates

### PR DESCRIPTION
**Pull request by Furtif**
* Update dependencies references
* Update SubModule https://github.com/Necrobot-Private/PokemonGO-Assets/pull/22
* Translation Updates @TheWizard1328 
* Try to fixes proxies issues (reverted not works, tested)
* Update POGOProtos to 0.67.2 (new pikachu costume for 2017 anniversary) https://github.com/Necrobot-Private/POGOProtos/pull/51

### CODE REVIEW

File: PoGo.NecroBot.Logic/State/Session.cs [hunk 1]
```diff
@@ -202,6 +202,31 @@ public bool ReInitSessionWithNextBot(Account bot = null, double lat = 0, double
             var session = TinyIoCContainer.Current.Resolve<ISession>();
 
             var nextBot = manager.GetSwitchableAccount(bot);
+
+            //var session = TinyIoCContainer.Current.Resolve<ISession>();
```
Comment (Lord-Haji) [inline PoGo.NecroBot.Logic/State/Session.cs]: Unused please remove

File: PoGo.NecroBot.Logic/State/Session.cs [hunk 2]
```diff
@@ -202,6 +202,31 @@ public bool ReInitSessionWithNextBot(Account bot = null, double lat = 0, double
...
+            if (XP == null) { XP = 0; }
+            long? SD = nextBot.Stardust;
+            if (SD == null) { SD = 0; }
+            var NLevelXP = nextBot.NextLevelXp;
+            if (nextBot.NextLevelXp == null) { NLevelXP = 0; }
+
+            Logger.Write($"Account changed to {Account}", LogLevel.BotStats);
+
+            if (session.LogicSettings.NotificationConfig.EnablePushBulletNotification == true)
+                PushNotificationClient.SendNotification(session, $"Account changed to", $"{Account}\n" +
```
Comment (Lord-Haji) [inline PoGo.NecroBot.Logic/State/Session.cs]: Should be Session in this context

File: PoGo.NecroBot.Logic/State/Session.cs (hunk 1 above)
Comment (Furtif) [inline PoGo.NecroBot.Logic/State/Session.cs]: Ok wait i am putting the good resources in rb

File: PoGo.NecroBot.Logic/State/Session.cs (hunk 2 above)
Comment (Furtif) [inline PoGo.NecroBot.Logic/State/Session.cs]: I do not know if it was @TheWizard1328  that modified it but I tested it and it's good

Comment (Lord-Haji) [inline PoGo.NecroBot.Logic/State/Session.cs]: I think session is undefined here.
Please check

Comment (Furtif) [inline PoGo.NecroBot.Logic/State/Session.cs]: https://github.com/Necrobot-Private/NecroBot/blob/master/PoGo.NecroBot.Logic/State/Session.cs#L202

**Comment by Furtif** [comment]
ready

File: PoGo.NecroBot.Logic/State/Session.cs (hunk 2 above)
Comment (Lord-Haji) [inline PoGo.NecroBot.Logic/State/Session.cs]: Ok

**Pull request merged**